packages/p2p-v2/src/pages/my-profile/screens/MyProfileStats/MyProfileStats.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { useActiveAccount } from '@deriv/api';
+import { useAdvertiserStats } from '../../../../hooks';
+import MyProfileStatsItem from './MyProfileStatsItem';
+import './MyProfileStats.scss';
+
+type MyProfileStatsProps = {
+    advertiserId?: string;
+};
+
+export const MyProfileStats = ({ advertiserId }: MyProfileStatsProps) => {
+    const [shouldShowTradeVolumeLifetime, setShouldShowTradeVolumeLifetime] = useState(false);
+    const [shouldShowTotalOrdersLifetime, setShouldShowTotalOrdersLifetime] = useState(false);
+    const { data } = useAdvertiserStats(advertiserId);
+    const { data: activeAccountData } = useActiveAccount();
+
+    if (!data || !activeAccountData) return <h1>Loading...</h1>;
+
+    // NOTE: This component is only visible for accounts with USD currency
+    if (activeAccountData?.currency !== 'USD') return <></>;
+
```

Comment:
should we have this check here? cause for p2p, we're supposed to block the entire app, if they are not using a USD account, i think we should handle this in our app.tsx tbh